src/avram/form_errors.cr

```diff
@@ -0,0 +1,12 @@
+module Avram::FormErrors
+  def errors
```

Comment:
Could you add a return type here? If you're not sure what it is you can make it String and then the compiler will tell you what it actually should be